docs/appendices/release-notes/5.8.5.rst

```diff
@@ -47,6 +47,11 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that would lead to wrong results being returned when trying to
+  match a column of type :ref:`CHAR<type-char>` with empty string literals, or
+  literals containing only whitespaces (``''``, ``'   '``, etc.) in the
```

Comment:
Isn't it irrelevant what chars are used?
```suggestion
  compare a column of type :ref:`CHAR<type-char>` with a column any character varying type 
  in the
```

Reply:
yes, I will rephrase, I realized when writing the commit message, but forgot.